Modules/Bibliographic/classes/FieldFilter/class.ilBiblFieldFilterFactory.php

```diff
@@ -29,7 +29,7 @@ class ilBiblFieldFilterFactory implements ilBiblFieldFilterFactoryInterface
     public function findById(int $id): \ilBiblFieldFilter
     {
         /** @noinspection PhpIncompatibleReturnTypeInspection */
-        return ilBiblFieldFilter::findOrFail($id);
+        return ilBiblFieldFilter::where(['id' => $id])->first();
```

Comment:
Is this change relevant for the refactoring? What happens if there is no entry with id `$id`? will lead to a fatal as well since then `null` instead of `\ilBiblFieldFilter` will be returned.